bin/import-resmap-collection

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+APP_PATH = File.expand_path('../../config/application',  __dir__)
+require_relative '../config/boot'
+require_relative '../config/environment'
+require_relative '../app/models/resmap/collection'
+require_relative '../app/models/resmap/site_normalizer'
+
+unless defined?(Rails)
+  require "logger"
+  require "time"
+end
+
+ResourceMap::Collection.origin = ENV.fetch("RESMAP_ORIGIN",  "http://web.resourcemap.lvh.me:3000")
+ResourceMap::Collection.username = ENV.fetch("RESMAP_USERNAME", "lradusky@manas.tech")
+ResourceMap::Collection.password = ENV.fetch("RESMAP_PASSWORD", "7214Lrad")
+collection_id = ENV.fetch("RESMAP_COLLECTION", "2").to_i
```

Comment:
**suggestion**: maybe we shouldn't have default credentials in the source code (even local ones) :)

```suggestion
ResourceMap::Collection.username = ENV.fetch("RESMAP_USERNAME")
ResourceMap::Collection.password = ENV.fetch("RESMAP_PASSWORD")
collection_id = ENV.fetch("RESMAP_COLLECTION")
```

I propose to still use `#fetch` as it will raise when the envvar doesn't exist with a helpful message.